python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1334,12 +1336,41 @@ def test_dataframe_hash_values(nrows, method):
     assert len(out) == nrows
     assert out.dtype == np.uint32
 
+    warning_expected = (
+        True if seed is not None and method not in {"murmur3"} else False
+    )
     # Check single column
-    out_one = gdf[["a"]].hash_values(method=method)
+    if warning_expected:
```

Comment:
Another alternative is to separate out the test for warning into a separate pytest and not use `seed` at all for this one.